libcxx/include/generator

```diff
@@ -0,0 +1,593 @@
+// -*- C++ -*-
```

Comment:
Does this complete the paper? If so can you update the `__cpp_lib_generator` feature-test macro?